docs/postmortems/2024-02-05-deprecated-icon-rendering.md

```diff
@@ -0,0 +1,113 @@
+---
+date: 2024-02-05
+authors: @literat
+---
+
+# Excessive deprecation logging while rendering Icon component
+
+**Summary**
+
+<!-- What is a one or two-line summary of the event that occurred? -->
+
+The `@lmc-eu/spirit-web-react@1.7.0` is shipped with a new deprecation warning about the `Icon` component.
```

Comment:
I don't see the problem here. Maybe the summary should already include the problem it caused. What do you think?

Reply:
Yup, you are right. Added.